seed_control_interface_service/settings.py

```diff
@@ -196,11 +196,24 @@
     'services.tasks.get_user_token': {
         'queue': 'mediumpriority',
     },
+    'services.tasks.fire_metric': {
+        'queue': 'metrics',
+    },
 }
 
+METRICS_REALTIME = [
+]
+METRICS_SCHEDULED = [
+]
+METRICS_SCHEDULED_TASKS = [
+]
```

Comment:
Do we need these? We don't have a `/metrics` endpoint in this service that uses them.